election-api-scala/app/model/PartyResult.scala

```diff
@@ -5,5 +5,7 @@ case class ConstituencyResult(id: Int, name: String, seqNo: Int, partyResults: S
 
 case class ApiResponse (error: String, message: String)
 
-
-case class Scoreboard(declared:Int)
\ No newline at end of file
+// TODO: this class should hold:
+//  - the overall winner (if there is one)
+//  - the seats that each party wins in Parliament
+case class Scoreboard(winner: String)
```

Comment:
Do you want *winner* parameter here? Feels like prejudicing the approach taken...

Reply:
I'd like that - the other languages don't lead the code approach like this, it's just the tests that imply a `winner` field needs to be grabbed somehow.